#### Задание 1.

Известно, что генеральная совокупность распределена нормально
со средним квадратическим отклонением, равным 16.
Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95,
если выборочная средняя M = 80, а объем выборки n = 256.

In [1]:
M = 80
std = 16
n = 256
alpha = 0.95

In [2]:
from scipy.stats import norm
import numpy as np

z = norm.ppf(alpha + (1-alpha)/2)
z

1.959963984540054

In [3]:
x1 = M-z*std/np.sqrt(n)
x2 = M+z*std/np.sqrt(n)
print(x1, x2)

78.04003601545995 81.95996398454005


Ответ: доверительный интервал при $\alpha=95\%$ от 78.04 до 81.96

#### Задание 2.

В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,
получены опытные данные:
6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1
Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,
оценить истинное значение величины X при помощи доверительного интервала, покрывающего это
значение с доверительной вероятностью 0,95.

In [4]:
X = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
alpha = 0.95
M = X.mean()
std = X.std(ddof=1)
n = len(X)
print(M, std, n)

6.590000000000001 0.4508017549014448 10


In [5]:
from scipy.stats import t
t1 = t.ppf(alpha + (1-alpha)/2, n-1)
t1

2.2621571627409915

In [6]:
x1 = M-t1*std/np.sqrt(n)
x2 = M+t1*std/np.sqrt(n)
print(x1, x2)

6.267515851415713 6.912484148584288


Ответ: доверительный интервал при $\alpha=95\%$ от 6.27 до 6.91

#### Задание 3.

Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр
оказался равным 17.5 мм, а дисперсия известна и равна 4 кв. мм.

In [7]:
M0 = 17
M = 17.5
n = 100
D = 4
alpha = 0.05
# H0: диаметр 17, М = М0
# H1: диаметр 17.5, М > М0

In [8]:
zt = norm.ppf(1-alpha)
zt

1.6448536269514722

In [9]:
zn = (M-M0)*np.sqrt(n)/np.sqrt(D)
zn

2.5

$Z_{n}>Z_{t}\Rightarrow$ верна гипотеза Н1 при $\alpha=5\%$

#### Задание 4.

Продавец утверждает, что средний вес пачки печенья составляет 200 г.
Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
Известно, что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [10]:
X = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
M0 = 200
alpha = 0.01

M = X.mean()
std = X.std(ddof=1)
n = len(X)
print(M, std, n)
# H0: вес 200, М = М0
# H1: М != М0

198.5 4.453463071962462 10


In [11]:
tt = t.ppf(1-alpha/2, n-1)
tt

3.2498355440153697

In [12]:
tn = (M-M0)*np.sqrt(n)/std
tn

-1.0651074037450896

$-t_{t}<t_{n}<t_{t}\Rightarrow$ верна гипотеза Н0 при $\alpha=1\%$